src/browser/rollbar.js

```diff
@@ -0,0 +1,269 @@
+var Client = require('../rollbar');
+var _ = require('../utility');
+
+var transforms = require('./transforms');
+var predicates = require('./predicates');
+var errorParser = require('./errorParser');
+
+function Rollbar(options, client) {
+  this.options = _.extend(true, defaultOptions, options);
+  var context = 'browser';
+  this.client = client || new Client(context, this.options);
+  this.init(this.client);
+}
+
+Rollbar.prototype.global = function(options) {
+  this.client.global(options);
```

Comment:
This does configure the global state stored on the Rollbar object instead of on the prototype instances.